App/Containers/PhotoViewerScreen.js

```diff
@@ -33,7 +33,7 @@ class PhotoViewerScreen extends React.PureComponent {
     return (<AsyncImage
       key={props.image.key}
       hash={props.image.hash}
-      progressiveLoad={props.image.progressiveLoad}
+      progressiveLoad='true'
```

Comment:
This is probably working for the wrong reason inside `AsyncImage` (would have the same result if set to `'false'`. Should be `progressiveLoad={true}`.

Reply:
ah, misread what you were saying there. fixed that up. but still hold off on rest for a minute